### ISSUE TITLE
Squared Brunt-Vaisala Frequency has 5 dimensions

### ISSUE THREAD

**Issue by gisi90** [Open-MSS/data-retrieval#25]
In the ml-NC file:

double n2(time, lev, lat, lon, lon_2) ;
                n2:_FillValue = NaN ;
                n2:units = "s^-2" ;
                n2:standard_name = "square_of_brunt_vaisala_frequency_in_air" ;

leads to a warning and n2 is skipped:

Adding potential temperature...
Adding potential vorticity...
Adding N2...
Adding first and second tropopause
separate sfc/ml
Creating pressure level file...
Warning (cdfCheckVars): 5 dimensional variables are not supported, skipped variable n2!
cdo    ml2pl: Processed 22764819 values from 17 variables over 7 timesteps [1.63s 192MB].

**Comment by joernu76** [comment]
Can you put the associated grib +settings.config files of one timestep somewhere, where I can access them?
https://gigamove.rwth-aachen.de/
?
I am not sure, where the second "lon_2" dimension is coming from. This doesn't happen in my test files.

**Comment by joernu76** [comment]
Looking at the code, this
```
diff --git a/bin/add_ancillary.py b/bin/add_ancillary.py
index e049e70..f35e6dd 100644
--- a/bin/add_ancillary.py
+++ b/bin/add_ancillary.py
@@ -234,7 +234,7 @@ def main():
     if option.n2:
         print("Adding N2...")
         xin["n2"] = brunt_vaisala_frequency_squared(my_geopotential_to_height(xin["zh"]), xin["pt"])
-        xin["n2"].data = xin["n2"].data.to(VARIABLES["n2"][1])
+        xin["n2"].data = xin["n2"].data.to(VARIABLES["n2"][1]).m
         xin["n2"].attrs["units"] = VARIABLES["n2"][1]
         xin["n2"].attrs["standard_name"] = "square_of_brunt_vaisala_frequency_in_air"
     if option.tropopause:
```
might help. However, no issue for my tests...

**Comment by joernu76** [comment]
Fixed by #26
